.github/actions/spell-check/line_forbidden.patterns

```diff
@@ -46,14 +98,53 @@
 # > In formal writing and where contractions are frowned upon, use `cannot`.
 # > It is possible to write `can not`, but you generally find it only as part of some other construction, such as `not only . . . but also.`
 # - if you encounter such a case, add a pattern for that case to patterns.txt.
-\b[Cc]an not\b
+\b[Cc]an not\b(?! only\b)
+
+# Should be `chart`
+(?i)\bhelm\b.*\bchard\b
+
+# Do not use `(click) here` links
+# For more information, see:
+# * https://www.w3.org/QA/Tips/noClickHere
+# * https://webaim.org/techniques/hypertext/link_text
+# * https://granicus.com/blog/why-click-here-links-are-bad/
+# * https://heyoka.medium.com/dont-use-click-here-f32f445d1021
+(?i)(?:>|\[)(?:(?:click |)here|link|(?:read |)more)(?:</|\]\()
+
+# Including "image of" or "picture of" in alt text is unnecessary.
+\balt=['"](?:an? |)(?:image|picture) of
+
+# Alt text should be short
+\balt=(?:'[^']{126,}'|"[^"]{126,}")
+
+# Should be `equals` to `is equal to`
+\bequals to\b
+
+# Should be `ECMA` 262 (JavaScript)
+(?i)\bTS\/EMCA\b|\bEMCA(?: \d|\s*Script)|\bEMCA\b(?=.*\bTS\b)
+
+# Should be `ECMA` 340 (Near Field Communications)
+(?i)EMCA[- ]340
+
+# Should be `fall back`
+\b(?<!\bon )fallback(?= to)\b
+
+# Should be `for`, `for, to` or `to`
+\b(?:for to|(?<!bind )to for)\b
 
 # Should be `GitHub`
 (?<![&*.]|// |\b(?:from|import|type) )\bGithub\b(?![{()])
 
 # Should be `GitLab`
 (?<![&*.]|// |\b(?:from|import|type) )\bGitlab\b(?![{()])
 
+# Should probably be `https://`...
+# Markdown generally doesn't assume that links are to urls
+\]\(www\.\w
+
+# Should be `https://`...
+(?<=<)http://(?!localhost|127|example|wiki\.powerdns\.com|www\.lmdb\.tech|BetterCGI\.com|ocsp\.int-x3\.letsencrypt\.org|=)(?:[^>]+)
```

Comment:
This is a custom pattern for this repository.
The `=` entry relates to an implementation detail (items matched by patterns are replaced by `=` before being reported by forbidden patterns) -- which feels more like a bug. I might try to fix it for the next version. -- The idea is that someone could write a forbidden pattern and essentially exempt something from the forbidden pattern by adding items for it in patterns, but given that one can use `?!` as done here, I'm not sure that's the right answer... I'm going to sleep on it for a while.